data_processing/integrity.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+
+# Checks the len of the dataframe
+def check_len(df, min):
+    if len(df) == 0:
+        raise Exception("Empty dataframe")
+    if len(df) < min:
+        raise Exception("Dataframe too short")
+
+# Checks the types in order of columns
+def check_types(df, types):
+    for i in range(len(df.columns)):
+        if df.dtypes[i] != types[i]:
+            raise Exception("Invalid type: " + df.columns[i] + " expected: " + str(types[i]))
+
+# Check if each column exists in the dataframe
+def check_columns(df, columns):
+    df_columns = df.columns
+    for i in range(len(df_columns)):
+        if not df_columns[i] == columns[i]:
```

Comment:
Denne vil fejle hvis ikke de er i rigtig rækkefølge
Her en en ændring som vil fix det
```suggestion
        if not df_columns[i] in columns:
```

Reply:
Jeg har ændret det hele til dictionary 📎